charset_normalizer/utils.py

```diff
@@ -1,4 +1,8 @@
-import unicodedata
+try:
+    import unicodedata2 as unicodedata
```

Comment:
Does this is the strict equivalent of what was done previously? I don't think so. methods like `isalpha` depend upon this, and doing that won't use the latest unicode data, I am wrong?

Reply:
By the looks of it, [`str.isalpha()`](https://github.com/python/cpython/blob/05162993fe62e7fa3acebdd0062586b9bf63d46a/Objects/unicodeobject.c#L12058-L12086) internally calls [`_PyUnicode_IsAlpha`](https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Objects/unicodectype.c#L288-L296), which calls [`gettyperecord`](https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Objects/unicodectype.c#L45-L59), which indexes into [`_PyUnicode_TypeRecords`](https://github.com/python/cpython/blob/bb3e0c240bc60fe08d332ff5955d54197f79751c/Objects/unicodetype_db.h#L4).

Unless importing `unicodedata2` also patches that interpreter-internal variable, then I don't think `isalpha` cares about `unicodedata2`.

Reply:
Okay then. Nice catch.